### PR TITLE
Billboard and Label tweaks

### PR DESCRIPTION
**Pull request by emackey**
This pulls out some feedback from the point-primitive branch (#2632) that was applicable to Billboards and Labels.

### CODE REVIEW

**Comment by emackey** [comment]
@pjcozzi please review this before looking at `point-primitive` again.  If this is approved I will integrate some changes from here to there.

File: Source/Scene/BillboardCollection.js [hunk 1]
```diff
@@ -1004,6 +1013,8 @@ define([
         boundingVolume.radius += size + offset;
     }
 
+    var writers = [];
```
Comment (pjcozzi) [inline Source/Scene/BillboardCollection.js]: Sorry to bother with local variable names, but can we include `scratch` in this like we do everywhere else?

**Comment by pjcozzi** [comment]
Wow, thanks for cleaning up all those tests.

> +434 −811

:)

**Comment by emackey** [comment]
This is ready.

**Pull request merged**